test/Repos.test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const { expect } = require('chai');
+require('chai').use(require('sinon-chai'));
+require('dotenv').config();
+const { repoFetch, reposFetch, orgRepoFetch, orgReposFetch } = require('./ReposMock');
+const nock = require('nock');
+const Repos = require('../src/models/Repos');
+const Repo = require('../src/models/Repo');
+
+describe('Repos model', () => {
+  const repos = new Repos('michaelmurray6298', 'skiski-app', 'ski-ski');
+  const allReposByUser = new Repos('michaelmurray6298', undefined, undefined);
```

Comment:
@ronanfitz to me it seems like these different patterns are probably deserving of their own `describe` blocks. Where possible, please refactor so that each call to `new Repos` lives in its own block. Thanks!